### DISCUSSION TITLE
LoRA has to be SDXL, is SD 1.5 LoRAs not possible?

### DISCUSSION THREAD

**Discussion by aytwong** [lllyasviel/Fooocus#1779] [Q&A] [answered]
Am I right in thinking SD 1.5 checkpoints can be use as refiners but LoRA has to be SDXL?

**Comment by JT-Gresham** [comment]
Nah...I'm using a LoRA right now that is SD 1.5 and it works great.

**Reply by aytwong** [reply]
Ah ok.. I couldn't get it to work.. Would you mind telling me how you used it please? My Lora does show up on the drop down list, but it wont do anything. i tried using a SD 1.5 model as a refiner as well...

**Reply by JT-Gresham** [reply, marked as answer]
Sure.  After you download it, you need to drop it into your lora folder...which I'm sure you did if it shows up for you in the dropdown.  After that, I just set the weight...then I use the trigger words in my prompt.  Like any other LoRA, it might not fire off consistently..especially if you've got multiple LoRAs going.  The kind of LoRA matters too, from what I've noticed. Some seem to use the weight to define intensity (sort-of) while others seem to use it as more of a priority.  Just how it seems to me but I might be wrong.

As far as the refiner...it should kick in when you tell it to with the slider at the bottom...but a refiner is just that. It will refine with images in its model slice if it can and do nothing if it can't (I think).

Not sure I gave you any info you didn't already know...but I hope this helped some.

**Reply by aytwong** [reply]
Thank you for taking time to answer.. I will try again with different LoRA thanks again :)

**Reply by eddyizm** [reply]
I've had mixed results with SD 1.5 LoRAs - Perhaps a list of working ones would be helpful. 
I also heard someone else mention that using a SD 1.5 as a refiner, then lets you activate all SD 1.5 loras - @JT-Gresham is this what you have experienced? Trying to nail down if that is a true statement and if so, now I need a SD 1.5 refiner lol.

**Reply by mashb1t** [reply]
@eddyizm every SD1.5 model can be used as refiner, but i've also had a very mixed if not bad experience with refiner and LoRA (compared to A1111 or ComfyUI), especially when switching early at 0.1 to enable the 1.5 based LoRA for as many steps as possible.

**Reply by eddyizm** [reply]
Really weird, just when I thought I figured it out I end up more lost.

**Reply by JT-Gresham** [reply]
@eddyizm 
Apologies for not replying sooner. It is my understanding that the refiner is just another checkpoint...which is different than a LoRA. The checkpoints are a collection of images that are used either to create images or refine them...a base set of actual images...while LoRAs are more like filters used to find suitable images dictated in your prompt.

That being said, the success rate is based on several factors, not least of all are the checkpoints used and how well they are trained by their creators.  So trying to find a good combination is part of the game.  Likewise, the LoRAs used can have a dramatic effect on the output you end up with...factors like weights and how the LoRA was generated play a significant role.  Both checkpoints and LoRAs will vary widely depending (number of epochs, keywords, prompt formatting...etc) and some are better than others.  The seeds used also play a big role.....sooo many variables to play with in order to get something close to what you want.

I've used SD 1.5 refiners with great success depending on what images I want to create...so yes, I don't restrict myself but I do experiment a LOT while trying to find good combinations.  Because of that, I don't think I could confidently give you a list of LoRAs...since they will behave differently depending on so many other factors.

I've added a lot of info here you might already have known just in case others who might be new to our AI imaging world read it...so don't take it like I'm being condescending. :)  I hope some of this helps in some way.  Cheers!
